include/nlohmann/byte_container_with_subtype.hpp

```diff
@@ -39,15 +39,15 @@ class byte_container_with_subtype : public BinaryType
         : container_type(std::move(b))
     {}
 
-    byte_container_with_subtype(const container_type& b, std::uint8_t subtype) noexcept(noexcept(container_type(b)))
+    byte_container_with_subtype(const container_type& b, std::uint8_t _subtype) noexcept(noexcept(container_type(b)))
```

Comment:
Please do not use a leading underscore here. Rather use `subtype_`.

Reply:
Ok, done